kotlin-analysis-api/src/main/kotlin/com/google/devtools/ksp/impl/symbol/kotlin/KSPropertyDeclarationImpl.kt

```diff
@@ -239,3 +233,11 @@ internal fun KaPropertySymbol.toModifiers(): Set<Modifier> {
     }
     return result
 }
+
+internal fun KSAnnotation.isValidOnProperty(): Boolean =
+    annotationType.resolve().declaration.annotations.none { metaAnnotation ->
+        metaAnnotation.annotationType.resolve().declaration.qualifiedName?.asString() == "kotlin.annotation.Target" &&
```

Comment:
Nit: maybe it's just me, but I prefer to avoid using constant as much as possible (in case of, e.g., typo or some other silly mistakes) :)

That said, this can be `Target::class.qualifiedName`? Same for `AnnotationTarget.PROPERTY` below. (But not necessarily in this change.)

Reply:
Will fix in later refactoring / polishing commits.